app/src/features/surveys/view/SurveyAnimals.tsx

```diff
@@ -58,6 +59,20 @@ const SurveyAnimals: React.FC = () => {
     setOpenAddCritterDialog((d) => !d);
   };
 
+  const pluralize = (str: string, count: number) =>
+    count > 1 || count === 0 ? `${count} ${str}'s` : `${count} ${str}`;
```

Comment:
`${count} ${str}'s` is technically wrong because of that apostrophe. What's wrong with using the `pluralize` helper in `Utils`?